src/applications/personalization/appointments/actions/utils.js

```diff
@@ -26,19 +31,27 @@ const getStagingID = facilityID => {
   return facilityID;
 };
 
+const isFutureAppointment = appointment => {
```

Comment:
time code always makes my head spin 🙃  would there need to be any reason to use utils from `date-fns-tz` here?

Reply:
ooh good point. looks like date-fns has a [`isFuture` helper](https://date-fns.org/v3.3.1/docs/isFuture) but it isn't included in their [functional programming submodule](https://date-fns.org/v3.3.1/docs/FP-Guide). i played around with it, and all of our tests still pass! 

i think i'm going to keep the `isFutureAppointment` function because that feels cleaner than having: 
```
const isUpcoming = isFuture(parseISO(appointment.attributes.start));
```
  
what do you think?

Reply:
Yea I think this helper function is fine, and like you said the alternative is a little verbose imo

Reply:
yep sounds good :) thanks for looking into that!